packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -44,7 +44,7 @@ governing permissions and limitations under the License.
 }
 .spectrum-Table-headCellContents {
   display: inline-block;
-  flex: 1 1 auto;
+  justify-self: start;
```

Comment:
o yeah, i guess it doesn't need to take up the whole area, probably don't even need justify-self start then since this won't every grow

Reply:
```suggestion
```